nixos/doc/manual/development/running-nixos-tests-interactively.xml

```diff
@@ -38,7 +38,11 @@ starting VDE switch for network 1
  </para>
 
  <para>
-  The machine state is kept across VM restarts in
-  <filename>/tmp/vm-state-</filename><varname>machinename</varname>.
+  You can keep the VMs states by using the <command>--keep-vm-state</command> flag.
+<screen>
+<prompt>$ </prompt>./result/bin/nixos-run-vms --keep-vm-state
+</screen>
+  The machine state will then be stored in the
+  <filename>/tmp/vm-state-</filename><varname>machinename</varname> directory.
```

Comment:
`/tmp` may be misleading, `$TMPDIR` may be clearer.

Reply:
:+1:

For the record:

```
~/Code/nixpkgs/nixos/doc(nin-delete-vm-state*) » ag "\/tmp"                                                                                                                                                                   
manual/release-notes/rl-1412.xml
406:     run <command>rm -rf /tmp/.vbox*</command>. The line <literal>imports = [
manual/release-notes/rl-1909.xml
228:     directory instead of <filename class="directory">/tmp</filename>. So
manual/release-notes/rl-2009.xml
102:        - allow MySQL to read from /home and /tmp directories when using <literal>LOAD DATA INFILE</literal>
manual/release-notes/rl-2003.xml
504:      If you rely on <literal>/tmp</literal> being shared with other services, explicitly override this by
manual/development/running-nixos-tests-interactively.xml
21:<prompt>&gt;</prompt> machine.succeed("touch /tmp/foo")
46:  <filename>/tmp/vm-state-</filename><varname>machinename</varname> directory.
manual/development/building-parts.xml
108:    /run/systemd/system/tmp-httpd.service

~/Code/nixpkgs/nixos/doc(nin-delete-vm-state*) » ag "\$TMPDIR"
< 0 match>                                                                                                                                                     
```

You might want to edit this tree-wide.